src/utils/send-request.js

```diff
@@ -37,7 +37,11 @@ function onRes (buffer, cb) {
     const isJson = res.headers['content-type'] &&
                    res.headers['content-type'].indexOf('application/json') === 0
 
-    log(res.req.method, `${res.req.getHeaders().host}${res.req.path}`, res.statusCode, res.statusMessage)
+    if (isNode) {
+      log(res.req.method, `${res.req.getHeaders().host}${res.req.path}`, res.statusCode, res.statusMessage)
+    } else {
+      log(res.url, res.statusCode, res.statusMessage)
```

Comment:
Is the method available or just the URL?

Reply:
I looked but did not see the method

Reply:
Fair enough

Reply:
It's all `POST` anyway..